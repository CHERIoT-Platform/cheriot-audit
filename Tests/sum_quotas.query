# Check that we can demangle a library export correctly.
--board inputs/sail.json -j inputs/test-suite.json -q 'sum([ data.rtos.decode_allocator_capability(c).quota | c = input.compartments[_].imports[_] ; data.rtos.is_allocator_capability(c) ])'

