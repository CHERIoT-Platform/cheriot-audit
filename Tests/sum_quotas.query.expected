{"expressions":[1070080]}
